bitrise.yml

```diff
@@ -276,6 +276,11 @@ workflows:
             $BITRISE_CACHE_DIR
             ./Carthage -> ./Carthage/Cachefile
     - deploy-to-bitrise-io@1: {}
+    - slack@3:
+        run_if: ".IsBuildFailed"
+        inputs:
+        - channel: "#firefox-ios-alerts"
```

Comment:
Should we do a `#focus-ios-alerts` channel?

Reply:
Sure, we can do that and ask for a webhook token for that channel. On it